tests/performance/pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[project]
```

Comment:
For packages the folder convention we follow in this repo it wouuld be

```
test/performace/
    - Makefile
    - README.md 
    - locust_files/
    - locust_settings/
	  - pyproject.tml
	  - requirements.txt
	  - src/
	       - __init__.py
	       - ... (other source files)
 ```

Reply:
Good point. Thanks. I have ajusted

Reply:
@bisgaard-itis  Well, you missed `src/` :-(
Anyway, sinceyou are using `pyproject.toml` perhaps you can help to implement https://github.com/ITISFoundation/osparc-simcore/issues/4433

Reply:
@pcrespov I tried using `src`, but it seems the name of the dir holding the source has to coincide with the project name (up to -/_). At least I was not able to get it to work otherwise and I can see we are doing the same elsewhere in the repo where we are using pyproject.toml.

Reply:
Sure, I would be happy to do https://github.com/ITISFoundation/osparc-simcore/issues/4433